astropy/cosmology/tests/test_parameter.py

```diff
@@ -70,16 +70,18 @@ def test_Parameter_init(self):
         assert parameter.format_spec == ""
         assert parameter.derived is False
         assert parameter.name is None
+        assert parameter._format == {}
```

Comment:
I wouldn't test private properties - it is an implementation detail that no formats is represented by `{}` instead of `None`

Reply:
✅